examples/solids/src/cloptions.c

```diff
@@ -257,7 +262,7 @@ PetscErrorCode ProcessPhysics(MPI_Comm comm, Physics phys, Units units) {
   // Define derived units
   units->Pascal = units->kilogram / (units->meter * PetscSqr(units->second));
 
-  // Scale E to GPa
+  // Scale E to Pa
```

Comment:
Do we want to update the README, on line 194? Where you describe the typical units in SI system, or it doesn't matter if that does not reflect the code?

Reply:
I think input and output should always be in SI. The `-units_*` should only have the effect of changing the internal scaling.

Reply:
Ok, but in the README it reads `Typical value in SI units`. Whereas I do understand that MPa or GPa are typical for Young's modulus, it is not technically neither its SI unit, nor the one now in the code. So I was just wondering if we wanted to update the table in the README or not

Reply:
Maybe I'm missing something, but I thought that we had everything in Pa now?

Reply:
Not in the [README](https://github.com/CEED/libCEED/blob/jeremy/solids-pressure/examples/solids/README.rst#nondimensionalization) line 194-195, if we care

Reply:
I took that to mean that a typical input for metal would be 100GPa, which is 1e11Pa? Like how cm is a typical unit for length but requires -units_meter 100? Maybe there is a better way to say that?

Reply:
Yes, but all I was saying is that the table column header reads `Typical value in SI units`. Sorry if I wasn't clear. Perhaps there is a better heading, or we express the Young's modulus in Pascal, rather than GPa in the table since that it is not really the SI unit (it would be Pascal since it has the units of pressure -- even though for solid materials it is most commonly used in GPa or MPa).

Reply:
Perhaps it would reduce confusion to use `-units_length 0.01` to use centimeters, etc, and just convert that to our internal representation.

The fact that we're still discussing this suggests that the table should just say `1e11 Pa` because it'll be unclear what a user should specify for `100 GPa`.

Reply:
That's what I was suggesting. Either change the table heading to not say SI units, or change the value in the table to not use GPa

Reply:
Updated the table. If we change the length scale scheme, it should be in a separate PR and we should do the same with fluids for consistency, imho. I'm fine with it as is or changed.